compiler/damlc/tests/daml-test-files/Text.daml

```diff
@@ -1,13 +1,13 @@
 -- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
 -- All rights reserved.
 
--- @INFO Evaluate
--- @INFO Evaluate
--- @INFO Use null
--- @INFO Use null
--- @INFO Use null
--- @INFO Use isNone
--- @INFO Use isNone
+-- @INFO range=41:9-41:17; Evaluate
```

Comment:
At least some of these hints here can just be addressed afaict instead of testing for them here.

Reply:
I appreciate that but can that not be done in another pass rather than doubling down on the work being done in this PR?

Reply:
Issue https://github.com/digital-asset/daml/issues/5642